src/views/Dashboard/components/TopTenTools/TopTenTools.js

```diff
@@ -93,6 +105,23 @@ function TopTenTools (props) {
   const { loading, error, data } = useQuery(GET_TOOL_EVENT_COUNT)
   const [toolTableData, setToolTableData] = useState({})
 
+  const focusOnOneColor = (name) => {
+    let data = {...toolTableData}
+    for (const tool in toolTableData) {
+      data[tool].color = tool !== name ? '#d0d0d0' : data[tool].color
```

Comment:
I think this is slightly clearer: 

```js
 for (const toolName in toolTableData) {
      data[toolName].color = toolName === name ? data[toolName].color :  '#d0d0d0'`
```
Also, it may be worth giving `'#d0d0d0'` an explicit name (is it grey?).